src/routes/Nav/Nav.tsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import { NavLink, Outlet } from 'react-router-dom';
+import '../../App.css';
+import Dropdown from '../../components/DropDown/DropDown';
+
+interface NavProps {}
+
+const Nav: React.FC<NavProps> = ({}) => {
+  return (
+    <div>
+      <nav>
+        <div className="logo">
+          <NavLink className={({ isActive }) => (isActive ? 'active' : 'activeNot')} to="/">
```

Comment:
use activeNot(with a different name) as default class and 'active' class optional